### PR TITLE
WD-12745 - rebrand /core/features/ota-updates

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

Rebrand /core/features/ota-updates page. No content was changed, layout only

## QA

- [demo link](https://ubuntu-com-14237.demos.haus/core/features/ota-updates)
- [figma](https://www.figma.com/design/5kDdBtPRqthyPz263YV91U/24.04-Ubuntu-Core-rebrand?node-id=1988-6058&t=PydHk0u9B31FTQKp-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma

## Issue / Card
[WD-12745](https://warthogs.atlassian.net/browse/WD-12745)

[WD-12745]: https://warthogs.atlassian.net/browse/WD-12745?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mattea-turic** [comment]
@lizzochek 

There's a bit of a spacing issue going on with the comps with the gifs, where there's a gap between the hr and the img container for the first two
Screenshot 2024-08-28 at 15 54 20

Same thing in the section below, and the first two gifs have a white background (though this is probably an assets problem, in which case if you send me a link to the gifs and I can try correct the background issue). I also feel that the last comp should have some whitespace at its vertical edges, but not sure if this is something that was intentional as I can't see the gifs in the figma file. This one seems to have different text in the figma too, do you know if there were changes made that don't reflect in the copydoc?
Screenshot 2024-08-28 at 15 57 13

**Comment by lizzochek** [comment]
@mattea-turic yes, I will need you to fix the gifs. Content changes were not made on this page, so I used the existing text on the page.

File: templates/core/features/ota-updates.html [hunk 1]
```diff
@@ -1,218 +1,201 @@
...
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width p-section--shallow">
+      <hr />
+      <h2>Reliability</h2>
+    </div>
+    <div class="row--25-75-on-large">
+      <div class="col">
+        <div class="p-equal-height-row">
```
Comment (mtruj013) [inline templates/core/features/ota-updates.html:52]: Looks like the first gif needs replacing as the top space doesn't match the other two
Screenshot from 2024-08-30 09-22-38

File: templates/core/features/ota-updates.html [hunk 2]
```diff
@@ -1,218 +1,201 @@
...
+                     alt="Update streams GIF" />
+              </div>
+            </div>
+            <div class="p-equal-height-row__item">
+              <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+              <h3 class="p-heading--5">Update streams from channels</h3>
+            </div>
+            <div class="p-equal-height-row__item">
+              <p>
+                Every snap can offer multiple streams of updates - called  channels - including the version and maturity -edge, beta, stable- of the snap . Switch to ‘3.2/stable’ and you know what you’ll be getting on that machine.
```
Comment (mtruj013) [inline templates/core/features/ota-updates.html:83]:
```suggestion
                Every snap can offer multiple streams of updates &mdash; called  channels &mdash; including the version and maturity -edge, beta, stable- of the snap . Switch to ‘3.2/stable’ and you know what you’ll be getting on that machine.
```

File: templates/core/features/ota-updates.html [hunk 3]
```diff
@@ -1,218 +1,201 @@
...
+      <h2>Ubuntu Core sets a new standard for Linux device updates, covering  the kernel, the OS and applications.</h2>
+    </div>
+    <div class="row--50-50">
+      <div class="col">
+        <p>
+          Transactional updates for reliability. Deltas minimize network traffic. Digital signatures guarantee integrity and provenance.
+        </p>
+        <p class="p-section--shallow">This is the future of smart things.</p>
+        <hr class="is-muted" />
+        <a class="p-button--positive" href="/internet-of-things/appstore">Get an IoT app store</a>
```
Comment (mtruj013) [inline templates/core/features/ota-updates.html:31]: These wrap strangely on medium, maybe an inline list with the `u-responsive-realign` class would be better here, but feel free to confirm with design. Difference here:
image

**Review by mtruj013 — APPROVED** [review]
Thanks @lizzochek , some minor comments, mostly visual

**Comment by mattea-turic** [comment]
@lizzochek Do you think you could add some padding for the "Safe operating system updates" component as the image is currently running edge-to-edge. I've also created a [version of it with padding](https://assets.ubuntu.com/v1/b2b8d9b4-ota-stack-case-with-padding.png), if you want to use that instead.
Screenshot 2024-08-30 at 17 41 42

And one tiny thing, could you add a `<br>` after "app store" pls?
Screenshot 2024-08-30 at 17 48 07

File: templates/core/features/ota-updates.html (hunk 1 above)
Comment (lizzochek) [reply templates/core/features/ota-updates.html:54]: @mattea-turic please check this out too

**Pull request merged**